python/test/unit/fem/test_element_integrals.py

```diff
@@ -8,11 +8,13 @@
 import random
 from itertools import combinations, product
 
+import dolfinx
 import numpy as np
 import pytest
 import ufl
-from dolfinx import (Constant, Function, FunctionSpace, VectorFunctionSpace,
-                     cpp, fem)
+from dolfinx.fem import (Constant, Function, FunctionSpace,
```

Comment:
General comment: Are `assemble_*` used frequently enough that they should be in the root namespace? I do not really see why eg. `Form`, `Function` and `UnitSquareMesh` are in the root namespace, but `assemble_*` are not.

Reply:
Equally, perhaps some things are currently not justified in the root namespace.

Reply:
`Form`, `Function`, etc, have been removed from the root namespace in this PR. They were in the root namespace for legacy reasons and just never got moved out.